script/Config/App.lua

```diff
@@ -14,8 +14,8 @@ Config.userInitFilename = "user.ini"
 Config.timeToResetToSplashscreen = 60
 
 Config.render = {
-    defaultResX    = 1600,
-    defaultResY    = 900,
+    defaultResX    = 2400,
+    defaultResY    = 1350,
```

Comment:
i think this should be 1080p tbh

Reply:
These are overridden by whatever each player has set in their user.ini file. The only ones who'll ever see these these default screen sizes (which I set to larger than 1600x900 because I got tired of resizing my screen across multiple test runs) are those using the test apps such as TestEcon and Quickstart.

But if it's a problem for other developers, I'll set defaultResY to 1080.

Reply:
yeah this is overridden, but the default res should be 1080 cause at first start it will be that resolution